ipc/cli/src/commands/subnet/set_federated_power.rs

```diff
@@ -0,0 +1,52 @@
+use std::str::FromStr;
```

Comment:
@mb1896 this is missing a license header, and CI is complaining. Please keep an eye on CI checks!

Reply:
Fixed. Thanks